int main()
{
    throw "Exception which will not be caught";

    return 0;
}
